framework/src/consumption/lib/opensearch/resources/lambda/opensearch-api/opensearch-api.mjs

```diff
@@ -30,10 +25,10 @@ export async function handler(event) {
 }
 
 const onUpdate = async (event) => {
-  const { path, body } = event.ResourceProperties;
+  const { path, body, method } = event.ResourceProperties;
   const secretsResolvedBody = await resolveSecrets(body);
```

Comment:
What is the timeout of the CR here? we might want to put it for some long time (maybe few minutes?) in case a user pass a big payload.